CandidateReranking.md

```diff
@@ -0,0 +1,369 @@
+| location | String | **FALSE** | Location requested in search.
+| should_use_radius | boolean | **FALSE** | Flag on the original search to determine use of radius.
+| radius | Integer | **FALSE** | Radius used in original search.
+| extracted_keywords | [Keyword[]](#keyword) | **FALSE** | Array of **Keyword** enrichments to the query string or `search_keywords` returned from SemanticSearchAPI.
+| parsed_input | ParsedInput | **FALSE** | **ParsedInput** Object comprised of a Map<String,String> as `input_to_extracted_keywords` and a list of [ParsedEntities](#ParsedInput)
+| filters | [Filters[]](#Filter) | **FALSE** | Array of filters used in the original query to search.
+
+#### ParsedInput
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| input_to_extracted_keywords | Map<String, String> | **FALSE** | Words extracted from `search_keywords` as the **Key** and their relavtive keywords identified by SemanticSearchAPI as the **Value**.
+| parsed_entities | ParsedEntity | **FALSE** | Entities parsed from `search_keywords`.
+
+#### ParsedEntity
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| entity_name | String | **FALSE** | Name of the entity.
+| is_phrase | Boolean | **FALSE** | Flags the entity as a phrase or not.
+| is_selected | Boolean | **FALSE** | Flags the entity as sleceted or not.
+
+#### Keyword
+Entites returned from the SemanticSearchAPI.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| name | String | **FALSE** | Name of the keyword, e.g. "Java".
+| type | String | **FALSE** | Type of the keyword, e.g. "skill".
+| weight | Integer | **FALSE** | Weight of the keyword as an int. \[0-100\]
+| relationships | [Relationship](#relationship) | **FALSE** | Array of relationship objects which enrich the keyword with additional entities.
+
+#### Relationship
+A `relationship` is composed of a **\<semantic enrichment type\>** and the associated set of entities which can be classified under that **\<semantic enrichment type\>**. Accepted values for the **\<semantic enrichment type\>** are: `job_titles`, `occupations`, `skills`, `job_level`, `text_kernel_related_keywords`, `interesting_keywords`, `related_keywords`, `related_keywords_recruiter`, `related_keywords_jobseeker`, `raw_job_titles_jobseeker`, `raw_job_titles_recruiter`, `extracted_kewords`, `custom_keywords`, `related_search_terms`, and `job_titles_recruiter`.
+
+| Param    | Type | Required | Description |
+|----------|------|----------|-------------|
+| **\<semantic enrichment type\>** | [Entity[]](#entity) | **FALSE** | Relationship to the keyword `name` value.
+
+#### Entity
+An `entity` is composed of a `name`, `weight`, and `selected` boolean. An `entity` enriches the original keyword with terms or phrases which share a close association. For example, if you were to have a `keyword` java, an enrichment entity might be java or programming.
```

Comment:
For the example here could we maybe pick a better example enrichment entity? I feel like saying "java" is an enrichment entity for "java" keyword doesn't get us much further in explaining what an entity is.